### DISCUSSION TITLE
Add Optimism Goerli

### DISCUSSION THREAD

**Discussion by klyap** [wevm/wagmi#743] [Ideas]
Since Optimism Kovan is being deprecated soon (August 15th, 2022) in favor of Optimism Goerli, can we add Optimism Goerli network to wagmi?

https://www.alchemy.com/overviews/migrate-from-kovan-to-goerli-on-optimism

**Comment by tmm** [comment]
That would be great! Feel free to open a PR.

**Reply by klyap** [reply]
Here it is! https://github.com/wagmi-dev/wagmi/pull/767
